README.md

```diff
@@ -200,6 +200,53 @@ KUBECOLOR_OBJ_FRESH="1m" kubecolor get po
 
 Default value is `0s`, it means is disabled.
 
+### Change Color Schema
+
+As of version `v0.3.0` it is now possible to use your own color scheme !
+
+To do so, define the environment variable `KUBECOLOR_CUSTOM_COLOR` as a SemiColums (`;`) delimited string, setting one or many of the available keys:
```

Comment:
My initial thought with #9 was an actual config file, such as `~/.kube/color.yaml`

That way we would have a lot of flexibility in what kind of options we allow. And could even allow users to add overrides on a per sub-command (or even sub-sub-command) basis.

So you could override thing like "use table coloring on `kubectl my-plugin get ...`" or "skip coloring on `kubectl version`"

But I don't think one excludes the other. Having per sub-command settings, and allowing mix of config file and environment variables, could be a nice future addition.

Reply:
when we have a basic functions for custom coloring, we'll add a custom coloring "per command"... and a config file for that. 
My fear is that it's going to complex to be useful, but it all depends on how we implement it... Maybe keep #9 for color and add a new issue for config file + plugins coloring ?

Reply:
BTW, one of the things I like with using the "default" color names, is that its based on the `terminal` color... so if you're using a theme (in iTerm, for ex), then `red` will be what was defined in the theme, not the "default" red... I have to check how libraries manages that.

Reply:
The library I linked does support regular ANSI codes too, so our default could be using the 16-color set, and then allow contributions of other themes (like maybe a Dracula theme, for example) that uses hex codes for more precise theming.